packages/Header/Header.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div id="header" pt3 pb3 mb3>
```

Comment:
Same comment about the attributes `ptX`, `pbX`... than https://github.com/mesg-foundation/mesg-components/pull/58/files#r336343955